environment.go

```diff
@@ -424,6 +427,75 @@ func (e *Environment) GetTime(key string) time.Time {
 	return casted
 }
 
+// List returns a sorted slice of keys.
+//
+// Example:
+//
+//	env := NewEnvironment(t)
+//	env.Put("key1", "v1")
+//	env.Put("key2", "v2")
+//	keys := env.List()
+func (e *Environment) List() []string {
+	opChain := e.chain.enter("List()")
+	defer opChain.leave()
+
+	e.mu.RLock()
+	defer e.mu.RUnlock()
+
+	keys := []string{}
+
+	for key := range e.data {
+		keys = append(keys, key)
+	}
+
+	sort.Slice(keys, func(i, j int) bool {
+		return keys[i] < keys[j]
+	})
+	return keys
+}
+
+// Match accepts a glob pattern and returns a sorted slice of
+// keys that match pattern.
+//
+// Example:
+//
+//	env := NewEnvironment(t)
+//	env.Put("key1", "v1")
+//	env.Put("key2", "v2")
+//	env.Put("abc", "v3")
+//	keys := env.Match("k*")
+func (e *Environment) Match(pattern string) []string {
```

Comment:
As for me, `Match` is a bit confusing here:

* env.Match call looks like we want to match contents of environment; but we instead want to find keys which match pattern
* in httpexpect, String.Match* method family work with regexps, and this one works with glob

Maybe rename it to Find or Glob? Unlike env.Match, it's hard to imagine how env.Glob can match anything besides keys, so Glob seems better.

Reply:
Changed to **Glob**